### ISSUE TITLE
Forward stdout from services 

### ISSUE THREAD

**Issue by amouat** [canonical/pebble#5]
Feature Request.

For integration with Kubernetes and Docker, it would be helpful if the stdout from services could be forwarded to the stdout for the main process, perhaps prefixed with the name of the process. To be concrete, from a test running pebble in a container, I get the following output:

```
$ docker run --rm --name test test
daemon.go:318: Started daemon.
daemon.go:289: POST /v1/services 2.114401ms 202
cmd_run.go:154: Started default services with change 1.
cmd_run.go:162: Exiting on terminated signal.
```

But I can't tell what the stdout from my service is. I'm sure there is a way I can get this from the pebble service, but what about providing something like the following as well:

```
$ docker run --rm --name test test
[PEBBLE] Started daemon.
[PEBBLE] POST /v1/services 2.114401ms 202
[PEBBLE] Started default services with change 1.
[A_SVC] My web app has started!
[B_SVC] Started B service
[A_SVC] Serving on port 80
[A_SVC] Shutting down
[PEBBLE] Exiting on terminated signal.
```

One issue is that if a pod is terminated, I really only have kubernetes logs to look at, which by default will only be the Pebble output and won't tell me about what's happening in my service.

Apologies if this is already in progress or if there's a flag I missed.

**Comment by benhoyt** [comment]
This will be fixed with https://github.com/canonical/pebble/pull/29

**Comment by axinojolais** [comment]
In the meantime, if you really need this, sending logs to /proc/1/fd/1 works

**Comment by benhoyt** [comment]
@axinojolais Clever!

**Comment by benhoyt** [comment]
This has been fixed in #29, which is finally merged! It will be included in Juju 2.9.3 which is going to be released next week. You can see the format at https://github.com/juju/juju/pull/13004 -- basically what you suggested, but also prefixed with a timestamp.
